SCons/Environment.xml

```diff
@@ -645,26 +649,26 @@ See also &f-link-env-AppendUnique;,
 </arguments>
 <summary>
 <para>
-Append new path elements to the given path in the
-specified external environment (&cv-link-ENV; by default).
-This will only add
-any particular path once (leaving the last one it encounters and
-ignoring the rest, to preserve path order),
-and to help assure this,
-will normalize all paths (using
-<function>os.path.normpath</function>
+Append path elements specified by <parameter>newpath</parameter>
+to the given search path string or list <parameter>name</parameter>
+in mapping <parameter>envname</parameter> in the &consenv;.
+Supplying <parameter>envname</parameter> is optional:
+the default is the execution environment &cv-link-ENV;.
+Optional <parameter>sep</parameter> is used as the search path separator,
+the default is the platform's separator (<systemitem>os.pathsep</systemitem>).
+A path element will only appear once.
+Any duplicates in <parameter>newpath</parameter> are dropped,
+keeping the last appearing (to preserve path order).
```

Comment:
unless delete_existing is True..

Reply:
just restating what was already there :)

Reply:
I just tried a testcase for the underlying function in `UtilTests.py`. It keeps the last if there are dupes in the added string/vector:
```py
        # check which is kept if there are dupes in new
        p1 = r'C:\dir\num\one'
        p3 = AppendPath(p1, r'C:\dir\num\two;C:\dir\num\three;C:\dir\num\two', sep=';')
        assert p3 == r'C:\dir\num\one;C:\dir\num\three;C:\dir\num\two', p3
```

Reply:
... should I commit these extra tests?  (note EnvironmentTests has its own, which seems a little excess since the "unit" is the function that does the real work, in UtilTests, I could even add there though...)

Reply:
I did add the changes, fwiw.